### PR TITLE
Add a rename dataset step

### PR DESCRIPTION
**Pull request by yvanlebras**

### CODE REVIEW

File: topics/climate/tutorials/panoply/tutorial.md [hunk 1]
```diff
@@ -87,7 +87,13 @@ In this tutorial, we will be using data from the [Copernicus Climate Data Store]
 >
 >    {% include snippets/change_datatype.md datatype="datatypes" %}
 >
-> 4. Add a tag to the dataset corresponding to `copernicus`
+> 4. Rename Datasets
+>
+>    As https://blablabla is not a beautiful name and can give error for some tools, it is a good practice to change dataset name by something more meaningfull, here removing `https://zenodo.org/record/3697454/files/` PATH as `.nc` extension for example.
```
Comment (bgruening) [inline topics/climate/tutorials/panoply/tutorial.md:92]: here renaming ... into ... for example?

Comment (yvanlebras) [reply topics/climate/tutorials/panoply/tutorial.md:92]: oups, yes ;) sorry

Comment (yvanlebras) [reply topics/climate/tutorials/panoply/tutorial.md]: better now ?

**Comment by bgruening** [comment]
@annefou :)

**Comment by yvanlebras** [comment]
No need to mention it I think as this is the case with several Galaxy tools ;) So maybe "just" propagating best practice around renaming dataset and so one is ok

**Comment by annefou** [comment]
Ok! Then that's perfect. Thank you.

**Pull request merged**